activation/handler.go

```diff
@@ -619,7 +633,7 @@ func (h *Handler) processATX(
 	atx.BaseTickHeight = baseTickHeight
 	atx.TickCount = leaves / h.tickSize
 
-	proof, err = h.storeAtx(ctx, atx)
+	proof, err = h.storeAtx(ctx, atx, watx.Signature)
```

Comment:
In the future we should separate the malfeasance detection from the actual storing of the ATX, but not as part of this PR 🙂